syntax-directory-refactor

kghamilton89 opened the pull request:
Format refactoring for `charts/syntax` directory. Sames changes as #114.

rodionos commented:
TO highlight required or no I would use the format we use in atsd docs: https://axibase.com/docs/atsd/api/data/series/get.html#path-parameters This reduces table width and allows for more content in description columns

kghamilton89 commented:
Commit `revert-tables` makes that change. I wanted to try something new, I thought it looked good.

kghamilton89 commented:
* This PR contains very few content changes, mostly just removing turns of phrase such as "Some thing <s>allows you to</s> imports new information."
* Added `import` setting to `configuration/README.md`
* Added link to `syntax/value_functions.md` from [Shared Settings](https://github.com/axibase/charts/blob/master/widgets/shared/README.md#value)
